LFPy/eegmegcalc.py

```diff
@@ -120,9 +120,9 @@ def calc_potential(self, p):
         pot_tan = self._calc_tan_potential(p_tan)
 
         pot_tot = pot_rad + pot_tan
-        mask = np.isnan(pot_tot)
-        return np.ma.masked_array(pot_tot, mask=mask)
-
+        # mask = np.isnan(pot_tot)
```

Comment:
@solveignaess, was there a reason for returning masked arrays?

Reply:
We included it to avoid plotting nan-values, in case the user gives a measurement location with a radius r that is smaller than the dipole location radius rz.
I like it better without the masked_arrays, though! I don't need it for my plots. I think you used it in an early version of the figure 2-script @espenhgn, so you might want to double check the input values.

Reply:
No issues with the figure_2.py script as far as I can tell.